### ISSUE TITLE
Proposal: get rid of fallback boot parameters

### ISSUE THREAD

**Issue by crcdng** [tidalcycles/vscode-tidalcycles#40]
Fallback boot parameters defined in https://github.com/tidalcycles/vscode-tidalcycles/blob/da95f23dea664ec2adb0849b89e0819a9f7df774/src/tidal.ts#L115 are silently applied when no path to `BootTidal.hs` is defined. This might appear convenient, however it can lead to [hard to track errors](https://github.com/tidalcycles/Tidal/issues/1014) as the boot parameter sets in this file and in `BootTidal.hs` almost certainly will continue to diverge. 

This problem should be made explicit to the user.

As a solution, I suggest to remove the fallback parameters. The plugin would then require to enter the path to `BootTidal.hs`. If Tidal is installed before the plugin it can pre-populate the path.

**Comment by yaxu** [comment]
Agreed. BootTidal.hs will again change a lot in the soon-to-be-released tidal v1.10.

**Comment by kindohm** [comment]
Agree, we want to make this change.
